octomap_server/src/OctomapServer.cpp

```diff
@@ -865,6 +887,17 @@ bool OctomapServer::resetSrv(std_srvs::Empty::Request& req, std_srvs::Empty::Res
   return true;
 }
 
+#ifdef STAMPED_OCTOMAP_SERVER
+void OctomapServer::onSetEpoch(const std_msgs::Time::ConstPtr& epoch) {
+  m_octree->removeStaleNodes(static_cast<uint32_t>(epoch->data.toSec()));
+}
+
+void OctomapServer::onSetDegradeThresh(const std_msgs::Duration::ConstPtr& thresh) {
+  m_octree->degradeOutdatedNodes(static_cast<uint32_t>(thresh->data.toSec()),
```

Comment:
Should `m_time_thresh` be set here before calling degradeOutdatedNodes?

Reply:
Good catch, this is like a one time degrade call kinda interesting idea on its own - but probably not what we want..  It may be okay to only set m_time_thresh/m_timeThresh and let the degrade call happen in the next insert pointcloud loop.